USAGE.md

```
@@ -514,6 +526,60 @@ Get the privacy type of a universe:
 |--------------|----------------------------------------------|------------|
 | `UniverseId` | The [universe ID](#universe-id) of the game. | Numeric ID |
 
+### userGames
+
+Provides a list of games a user has created in the [JSON format](#Handling-JSON-data).
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
Expected: #handling-json-data; Actual: #Handling-JSON-data

(MD051, link-fragments)